cmd/ocm-backplane/console/console.go

```diff
@@ -129,6 +130,65 @@ func init() {
 	)
 }
 
+func checkAndFindContainerURL(containerName string, containerEngine string) (err error) {
+	existCheckArgs := []string{
+		"container",
+		"ps",
+		"--filter",
```

Comment:
The `--filter` here is specific to podman I suppose? For someone using a mac, we use docker and it errors out with:
```
➜ ./ocm-backplane console -b                                                                                    
INFO[0000] Using container engine docker                
Error response from daemon: Invalid filter '"name'
```
Can you try splitting it in two checks based on the container engine?